dotnet/ios/ios13/DisablingPullingDownASheet/DisablingPullingDownASheet/EditViewController.cs

```diff
@@ -0,0 +1,130 @@
+﻿/*
+See LICENSE folder for this sample’s licensing information.
+
+Abstract:
+A view controller for editing saved text. Allows cancellation and saving with standard bar button items, as well as with the pull to dismiss gesture.
+*/
+
+namespace DisablingPullingDownASheet;
+public partial class EditViewController : UIViewController, IUITextViewDelegate, IUIAdaptivePresentationControllerDelegate
+{
+	public EditViewController(IntPtr handle) : base(handle)
+	{
+	}
+
+	#region Model
+
+	string originalText = "";
+	public string OriginalText
+	{
+		get => originalText;
+		set
+		{
+			EditedText = value;
+			originalText = value;
+		}
+	}
+
+	string editedText = "";
+	public string EditedText
+	{
+		get => editedText;
+		set
+		{
+			editedText = value;
+			ViewIfLoaded?.SetNeedsLayout();
+		}
+	}
+
+	public bool HasChanges { get => OriginalText != EditedText; }
```

Comment:
You can rewrite this without the get in the new C#
```suggestion
	public bool HasChanges  => OriginalText != EditedText;
```